src/main/java/uk/ac/ebi/subs/validator/core/handlers/StudyHandler.java

```diff
@@ -42,6 +42,10 @@ public StudyHandler(@NonNull AttributeValidator attributeValidator, @NonNull Ref
 
         DataType dataType = getDataTypeFromRepository(envelope.getDataTypeId());
 
+        if (envelope.getProject() == null) {
+            return new ArrayList<>();
+        }
```

Comment:
I am not very familiar with validation side of things and so it may sound like a stupid question. But is it ok skip validation of the study altogether if the submission envelope is missing project information?

Reply:
We already discussed it on slack, that we already have exactly the same validation in JSON schema. For ENA study data type it stays there, for the EVA study data type it won't be there. So basically we could completely remove this validation later.